eli5/__init__.py

```diff
@@ -73,6 +72,12 @@
 except ImportError:
     # lightgbm is not available
     pass
-except OSError:
```

Comment:
I guess this check was removed by error. Could you put it back?

Reply:
I have seen that you added this `except` to all import trials in the file. Not sure this is right. Probably `lightgbm` included this `except` because in case of bad installation is failing improperly, but for the rest, I think is better to keep them as they were. Could you please leave `OSError` check only for `lightgbm` as it was before?